packages/ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@superdispatch/ui",
-  "version": "0.10.3",
+  "version": "0.10.4",
```

Comment:
@normurod72 You should not change version manually. It is done via `yarn run release`